source/agora/consensus/validation/Block.d

```diff
@@ -673,14 +673,14 @@ version (unittest)
             },
             prev_timestamp, (curr_timestamp == ulong.max) ? block.header.timestamp : curr_timestamp,
             block_timestamp_tolerance_dur,
-            (const ref Transaction[] tx_set, const ref uint[] missing_validators)
+            (in Transaction[] tx_set, in uint[] missing_validators)
             {
-                return cast(Transaction[]) [];
+                return (Transaction[]).init;
```

Comment:
Are the parentheses needed, although the `cast` has been removed?

Reply:
Yes because it's an array. `Transaction.init` parses fine, but `Transaction[].init` will yield a compiler error.